generator/moreschemavalidation_fixtures_test.go

```diff
@@ -75,28 +75,28 @@ func initFixture2116() {
 	}, append(todo, "omitempty"), noLines, noLines)
 
 	flattenRun.AddExpectations("object_with_arrays.go", []string{
-		"Array0 ArrayWithDefault `json:\"array0,omitempty\"`",
-		"Array1 []string `json:\"array1\"`",
-		"Array11 []string `json:\"array11,omitempty\"`",
-		"Array12 []string `json:\"array12\"`",
-		"Array2 ArrayWithOmitEmpty `json:\"array2,omitempty\"`",
-		"Array3 ArrayWithNoOmitEmpty `json:\"array3\"`",
+		"Array0 ArrayWithDefault `json:\"array0,omitempty\" xml:\"array0\"`",
+		"Array1 []string `json:\"array1\" xml:\"array1\"`",
+		"Array11 []string `json:\"array11,omitempty\" xml:\"array11\"`",
+		"Array12 []string `json:\"array12\" xml:\"array12\"`",
+		"Array2 ArrayWithOmitEmpty `json:\"array2,omitempty\" xml:\"array2\"`",
+		"Array3 ArrayWithNoOmitEmpty `json:\"array3\" xml:\"array3\"`",
 	}, todo, noLines, noLines)
 
 	flattenRun.AddExpectations("object_with_nullable_false.go", []string{
-		"Data interface{} `json:\"Data,omitempty\"`",
+		"Data interface{} `json:\"Data,omitempty\" xml:\"Data\"`",
 	}, todo, noLines, noLines)
 
 	flattenRun.AddExpectations("object_with_nullable_true.go", []string{
-		"Data interface{} `json:\"Data,omitempty\"`",
+		"Data interface{} `json:\"Data,omitempty\" xml:\"Data\"`",
 	}, todo, noLines, noLines)
 
 	flattenRun.AddExpectations("object_with_omitempty_false.go", []string{
-		"Data interface{} `json:\"Data,omitempty\"`",
+		"Data interface{} `json:\"Data,omitempty\" xml:\"Data\"`",
```

Comment:
Why don't we have support for `omitempty` in the case of XML serialization?

Reply:
I have now added omitempty support. As stated above, the reasons for me to not add it in the first place were that
* I proposed to first think about refactoring the json struct tag templates. I have now decided to put the entire logic for xml struct tags into the buildXMLNameWithTags function. This is a deliberate deviation from the json struct tags implementation.
* omitempty is only relevant for serializing, not for deserializing (cf. also https://golang.org/pkg/encoding/xml/#Marshal). I have now additionally manually tested the generation of a server which produces xml. Am I missing something or aren't there any integration tests that actually test serializing and deserializing requests and responses?